genie-web/src/main/java/com/netflix/genie/web/spring/autoconfigure/RetryAutoConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *
+ *  Copyright 2019 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.web.spring.autoconfigure;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.retry.annotation.EnableRetry;
+
+/**
+ * Auto configuration which enables {@link org.springframework.retry.annotation.Retryable} for method calls on beans.
+ *
+ * @author tgianos
+ * @since 4.0.0
+ */
+@Configuration
+@EnableRetry
+public class RetryAutoConfiguration {
+}
```

Comment:
I'm assuming this won't have any effect on the existing `retryTemplate` we have around?